wasm/text/decoder.go

```diff
@@ -331,7 +361,7 @@ func (p *moduleParser) addFunctionName(name string) {
 // Ex. If there is no signature `(import "" "main" (func))`
 //                     calls onImportFunc here ---^
 func (p *moduleParser) parseImportFunc(tok tokenType, tokenBytes []byte, line, col uint32) (tokenParser, error) {
-	idx := wasm.Index(len(p.module.ImportSection))
+	idx := p.module.SectionSize(wasm.SectionIDImport)
```

Comment:
I have in-flight work to make an import parser. Due to this `module.SectionSize` function, I can decouple the parser from needing a reference to the module via a function type like... `func(SectionID) uint32`